### PR TITLE
9 retrieve case data for user

### PR DESCRIPTION
**Pull request by yrao6597**
based on the schema for the cases data that was modified in Youssef's PR

### CODE REVIEW

File: Backend/app/controllers/customer.controller.js [hunk 1]
```diff
@@ -14,6 +14,21 @@ const getAllUsers = ((req, res) => {
     });
 });
 
+const getCases = ((req, res) => {
+    const clientId = req.query.clientId;
```
Comment (youssefsoli) [inline Backend/app/controllers/customer.controller.js]: Use `user_id` instead of `clientId` for consistency throughout the API

Comment (yrao6597) [reply Backend/app/controllers/customer.controller.js]: Good point! Thanks Youssef

File: Backend/app/controllers/customer.controller.js [hunk 2]
```diff
@@ -15,10 +15,10 @@ const getAllUsers = ((req, res) => {
 });
 
 const getCases = ((req, res) => {
-    const clientId = req.query.clientId;
+    const userId = req.query.userId;
```
Comment (youssefsoli) [inline Backend/app/controllers/customer.controller.js]: `user_id`

Comment (yrao6597) [reply Backend/app/controllers/customer.controller.js]: code updated

**Pull request merged**